src/errors.h

```diff
@@ -2296,6 +2296,8 @@ EXTERN char e_not_possible_to_change_sign_str[]
 #ifdef FEAT_VIMINFO
 EXTERN char e_cant_rename_viminfo_file_to_str[]
 	INIT(= N_("E886: Can't rename viminfo file to %s!"));
+EXTERN char e_warning_registers_partially_written_to_viminfo[]
+	INIT(= N_("E2000: Warning: registers only partially written to viminfo, increase viminfo size"));
```

Comment:
If I understood your description correctly, then this is not an error, but a warning.
Like this:
https://github.com/vim/vim/blob/68f8cfb7efa79510a01dd55d35cb5c7456a7261f/src/buffer.c#L2278
Accordingly, the number should not start with the letter "E", but with the letter "W". The last warning number was "W22".
> Note 2: What error number should be applied to the new warning (I set it to 2000)?
> 

>

Reply:
Oh okay and why are there Warnings under src/errors.h?